packages/jade/tests/match.html

```diff
@@ -118,4 +118,9 @@
 <template name="match-html-specialMarkdownComponent">{{#markdown}}This is my email: <address@example.com>
 {{/markdown}}</template>
 
-<template name="match-html-specialMarkdownComponentWithHelper">{{#markdown}}This is my email: {{email}}{{/markdown}}</template>
+<template name="match-html-specialMarkdownComponentWithHelper">{{#markdown}}This is my email: {{email}}
+{{/markdown}}</template>
+
+<template name="match-html-anonymousIfHelper">{{#if _jade_line257}}hello world{{/if}}</template>
```

Comment:
I removed the tests. I believe runtime.js is enough for testing to see if the code works as intended (maybe more test cases could be added there to see if there are important not handled edge cases). What do you think?